theano/sandbox/cuda/blas.py

```diff
@@ -417,13 +419,14 @@ def __hash__(self):
         return hash(type(self)) ^ hash(self.inplace)
 
     def __setstate__(self, dct):
-        inplace = dct.get('inplace', True)
-        if inplace:
-            self.destroy_map = {0: [0]}
-        self.inplace = inplace
+        self.__dict__.update(dct)
 
-    def __getstate__(self):
-        return dict(inplace=self.inplace)
+        # Correctly reload older pickles where _op_use_c_code and
+        # destroy_map were not saved
+        if '_op_use_c_code' not in self.__dict__:
+            self._op_use_c_code = theano.config.cxx
```

Comment:
According to [the comments in `Op.__new__`](https://github.com/Theano/Theano/blob/master/theano/gof/op.py#L768-L774), this is not optional and all Ops should have it.